test/components/project/user/list.spec.js

```diff
@@ -155,20 +155,19 @@ describe('ProjectUserList', () => {
         });
     });
 
-    it('renders the select correctly for the current user', () => {
+    it('renders the select correctly for the current user', async () => {
```

Comment:
Making this async because that will make things easier in the PR for instant tooltips.